pkg/controller/cheinstallation/cheinstallation_controller_test.go

```diff
@@ -288,18 +289,39 @@ func TestCheInstallationController(t *testing.T) {
 			HasConditions(SubscriptionFailed(errMsg))
 	})
 
+	t.Run("should fail to create operator group when namespace is not active", func(t *testing.T) {
+		//given
+		cheInstallation := NewInstallation()
+		cheOperatorNS := cheInstallation.Spec.CheOperatorSpec.Namespace
+		cl, r := configureClient(t, cheInstallation,
+			newCheNamespace(cheOperatorNS, v1.NamespaceTerminating))
+		request := newReconcileRequest(cheInstallation)
+
+		// when
+		result, err := r.Reconcile(request)
+
+		//then
+		require.NoError(t, err) // no error is reported...
+		assert.True(t, result.Requeue)
+		assert.Equal(t, 3*time.Second, result.RequeueAfter)
+		AssertThatOperatorGroup(t, cheOperatorNS, OperatorGroupName, cl).
+			DoesNotExist()
+	})
+
 }
 
 func TestCreateOperatorGroupForChe(t *testing.T) {
 
+	logf.SetLogger(zap.Logger(true))
```

Comment:
we don't need this - it should be activated in the `init` function https://github.com/codeready-toolchain/toolchain-operator/pull/21/files#diff-5b9ac2c662abb9c03aabe8163fc16509R33-R36

Reply:
right, I forgot about that one :/ 
removed in e5acc23